### DISCUSSION TITLE
Angular library using Tailwind

### DISCUSSION THREAD

**Discussion by KoenVerhulst** [tailwindlabs/tailwindcss#8783] [Help]
Hi,

It seems to be a hot topic in the discussions, however many threads remain unanswered:

I'm currently trying to create a company wide (angular 14) UI component library. We did this before in angular 9 using simple .scss.
Our new UI uses tailwind (tailwind utitlities, @apply in custom css, ...). We would like to create a library containing the UI.

We are able to publish the lib in our private registry but as soon as we use it in an application functionality like @apply (from the libary) is not working. Browser just notes that "@apply" is an unknown element.

My question:

- Is there any way to get this working?
- Are there any plans to get this working?
- What are the alternatives? Remove/rewriting all the @apply functionality would be a big work.

Thanks in advance.

Koen

**Comment by CharlyRipp** [comment]
Duplicate question that I just commented on -- https://github.com/tailwindlabs/tailwindcss/discussions/8002#discussioncomment-4278737
